app/feature/feature-chat/src/main/kotlin/com/hedvig/android/feature/chat/ChatLoadedScreen.kt

```diff
@@ -628,16 +669,47 @@ private fun ChatBubble(
     },
     uiChatMessage = uiChatMessage,
     chatItemIndex = chatItemIndex,
-    modifier = modifier,
+    modifier = modifier.semantics(mergeDescendants = true) {
+      contentDescription = description
+    },
   )
 }
 
+@Composable
+private fun getMessageDescription(chatMessage: CbmChatMessage?): String {
+  return chatMessage?.let {
+    val context = LocalContext.current
+    val sender = stringResource(
+      when (it.sender) {
+        Sender.HEDVIG -> R.string.CHAT_SENDER_HEDVIG
+        Sender.MEMBER -> R.string.CHAT_SENDER_MEMBER
+      },
+    )
+    val time = formatInstantForTalkBack(context, it.sentAt)
+    when (it) {
+      is ChatMessageFile -> stringResource(R.string.TALKBACK_CHAT_MESSAGE_FILE, time, sender, it.mimeType)
+      // At [%s:formattedInstant], [%s:sender] sent a file with type: [%s:mimeType], double tap to open.
+      is ChatMessageGif -> stringResource(R.string.TALKBACK_CHAT_MESSAGE_GIF, time, sender)
+      // At [%s:formattedInstant], [%s:sender] sent a GIF picture.
+      is CbmChatMessage.ChatMessageText -> stringResource(R.string.TALKBACK_CHAT_MESSAGE_TEXT, time, sender, it.text)
+      // "at $instant, $sender said: $it.text."
+      is FailedToBeSent.ChatMessageMedia -> stringResource(R.string.TALKBACK_CHAT_FAILED_MEDIA, time)
+      // "at $instant, you tried to send a media file, but it failed. Double tap to try sending again."
+      is FailedToBeSent.ChatMessagePhoto -> stringResource(R.string.TALKBACK_CHAT_FAILED_PHOTO, time)
+      // "at $instant, you tried to send a photo, but it failed. Double tap to try sending again."
+      is FailedToBeSent.ChatMessageText -> stringResource(R.string.TALKBACK_CHAT_FAILED_TEXT, time, it.text)
```

Comment:
These too. Will we get help for translations on all of these things I wonder, or will we do our own translations :D